src/app/app.module.ts

```diff
@@ -51,23 +51,26 @@ import { environment } from '../../src/environments/environment';
     FilterModule,
     FormsModule,
     ToolbarModule,
-    ToastNotificationModule
+    EmptyStateModule
   ],
   providers: [
     AuthenticationService,
     Broadcaster,
+    { provide: HTTP_INTERCEPTORS, useClass: AuthInterceptor, multi: true },
+    { provide: HTTP_INTERCEPTORS, useClass: CacheInterceptor, multi: true },
     { provide: AUTH_API_URL, useValue: environment.authApiUrl },
     { provide: SSO_API_URL, useValue: environment.ssoApiUrl },
     { provide: WIT_API_PROXY, useValue: environment.witApiUrl },
     { provide: ADMIN_API_URL, useValue: environment.adminApiUrl },
+    { provide: AUTH_API_URL, useValue: 'https://auth.prod-preview.openshift.io/api/' },
```

Comment:
@nemesis09 why is this again setting values for api urls directly?

Reply:
Sorry.
Missed it in cleanup. It's fixed.